setup.py

```diff
@@ -59,7 +59,7 @@ def find_version(*file_paths):
     install_requires=install_requires,
     extras_require={
         "dev": ["black", "twine", "pre-commit"],
-        "docs": ["ipython", "ipykernel", "sphinx<3.0.0", "sphinx_rtd_theme", "nbsphinx", "m2r"],
+        "docs": ["ipython", "ipykernel", "sphinx", "sphinx_rtd_theme", "nbsphinx", "m2r2"],
```

Comment:
While we're cleaning up the dependencies, we could probably get rid of the docs `extras_require`, since we exclusively use `docs/requirements.txt`. So I would propose just deleting this line.